frontend/app/equity/dividend_computations/[id]/confirm/page.tsx

```
@@ -0,0 +1,296 @@
+  );
+}
+
+export default function ConfirmDividendComputationPage() {
+  return (
+    <EquityLayout>
+      <Suspense fallback={<div>Loading...</div>}>
+        <ConfirmDividendComputationContent />
+      </Suspense>
+    </EquityLayout>
+  );
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)